src/init.js

```diff
@@ -29,13 +29,21 @@ export function initApp (queryString = window.location.search) {
   if (queryString.length === 0) {
     updateURL(mapView)
   }
+
   // Initializing lat/lng and zoom
   store.dispatch(recenterMap(coordinates, mapView.zoom))
   // Initializing map search bar
   store.dispatch(setLocation(coordinates, label))
   // Initializing dates
   store.dispatch(setDate(date.startDate, date.endDate))
 
+  // Initializing analysis view name
+  if (object.viewName) {
+    const viewName = object.viewName
+    document.title = viewName + ' | OpenTraffic Analyst UI '
```

Comment:
Yeah, that's a good point. I do think it would be preferable to set document.title from just one module. So both the component constructor (or `componentDidMount()`) and `handleSubmit()` can call a single function/method that changes the document.title.